app/assets/scripts/map.js

```diff
@@ -97,37 +130,62 @@ function create_marker(resource){
   async.each(values,
     function(value, callback){
       markerToAdd.addListener('click', function() {
+        $("#map").show();
+        $("#resource-info").hide();
+
         var contentDiv = document.createElement('div');
-        var strong = document.createElement('strong');
+        var resName = document.createElement('strong');
+        $(resName).html(resource.name);
         var br = document.createElement('br');
-        $(strong).html(resource.name);
-        $(contentDiv).append(strong, br, resource.address);
+
+        // information pane to replace map
+        // TODO: refactor how the page elements are generated
```

Comment:
There has to be a better way than this right? I hate the way we are creating html. Is there some light-weight client-side framework for generating html in js?